release-notes/opensearch-geospatial.release-notes-2.3.0.0.md

```diff
@@ -0,0 +1,7 @@
+## Version 2.3.0.0 Release Notes
+
+Compatible with OpenSearch 2.3.0
+
+### Refactoring
+
+* Fix typo errors ([#125](https://github.com/opensearch-project/geospatial/pull/125))
```

Comment:
This PR is related to XYShape. I think we don't need to include this for 2.3 release